datasetinsights/datasets/synthetic.py

```diff
@@ -247,3 +289,11 @@ def exists(capture_file):
         keep_mask = catalog.filename.apply(exists)
 
         return catalog[keep_mask]
+
+    @staticmethod
+    def is_synthetic_dataset_files_present(dataset_directory):
```

Comment:
But how do we check for empty folder? because we would unpack dataset archive file in the same directory and what if the archive file is empty then the dataset archive file would still be there and give False for empty directory check.

Reply:
```
from pathlib import Path 

def exists():
    p = Path(data_path) / "folder_name"
    return p.exits() and len(p.glob("**/*")) > 0
```